content/help/faq/distributed-tracing/how-distributed-tracing-works.md

```diff
@@ -0,0 +1,37 @@
+---
+title: How does Distributed Tracing work with Istio?
+weight: 0
+---
+
+Istio integrates with distributed tracing systems in two different ways: Envoy-based and Mixer-based tracing integrations.
+
+For Envoy-based tracing integrations, Envoy (the sidecar proxy) sends tracing information directly to tracing backends on behalf of the applications being proxied.
```

Comment:
seem there should be a sub heading here each for envoy-based and mixer-based

Reply:
Broke these out into separate entries. Let me know what you think.